core/drag_target.js

```diff
@@ -33,6 +33,19 @@ const {Rect} = goog.requireType('Blockly.utils.Rect');
  * @alias Blockly.DragTarget
  */
 class DragTarget {
+  /**
+   * Constructor for DragTarget. It exists to add the id property and should not
```

Comment:
If this isn't meant to be constructed directly, should it be tagged `@abstract`?